sources/Inputs/src/Configuration.cpp

```diff
@@ -124,5 +126,51 @@ Configuration::Configuration(const boost::filesystem::path& filepath) {
     throw Error(ErrorConfigFileRead, e.what());
   }
 }
+
+void
+Configuration::updateChosenOutput(
+  const boost::property_tree::ptree& tree,
+#if _DEBUG_
+  dfl::inputs::Configuration::SimulationKind
+#else
+  dfl::inputs::Configuration::SimulationKind simulationKind
+#endif
+  ) {
+#if _DEBUG_
+  chosenOutputs_.insert(dfl::inputs::Configuration::ChosenOutputEnum::STEADYSTATE);
+  chosenOutputs_.insert(dfl::inputs::Configuration::ChosenOutputEnum::CONSTRAINTS);
+  chosenOutputs_.insert(dfl::inputs::Configuration::ChosenOutputEnum::TIMELINE);
```

Comment:
Maybe I am wrong but I though TIMELINE was never activated by default?

Reply:
According to what I was told to implement, TIMELINE is always activated by default. Maybe @marcochiaramello can confirm ?

Reply:
it is fine for me to dump all when in debug mode